### PR TITLE
Feature/update inspector

### PR DESCRIPTION
**Pull request by JulienChampagnol**

### CODE REVIEW

File: .github/workflows/udpdate.yml [hunk 1]
```diff
@@ -17,16 +17,34 @@ jobs:
     - name: "Skip if no new version"
       uses: Geode-solutions/actions/need-update@master
       id: update
-    - name: Stop
-      if: !steps.update.outputs.update
-      run: exit 0
+      with:
```
Comment (BotellaA) [inline .github/workflows/udpdate.yml:20]: You may a merge. I hope you did not change this file

**Review by BotellaA — APPROVED** [review]

File: .github/workflows/udpdate.yml (hunk 1 above)
Comment (JulienChampagnol) [reply .github/workflows/udpdate.yml:20]: I think i created this branch from the develop branch and not the master, that's why

Comment (BotellaA) [reply .github/workflows/udpdate.yml:20]: So update from master before to avoid strange diff to review

Comment (JulienChampagnol) [reply .github/workflows/udpdate.yml:20]: That's what I did

**Pull request merged**